relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt

```diff
@@ -0,0 +1,58 @@
+package xyz.qwewqa.relive.simulator.core.presets.dress.back.space
+
+import xyz.qwewqa.relive.simulator.core.presets.condition.BackOnlyCondition
+import xyz.qwewqa.relive.simulator.core.presets.dress.generated.dress3020009
+import xyz.qwewqa.relive.simulator.core.stage.actor.ActType
+import xyz.qwewqa.relive.simulator.core.stage.autoskill.new
+import xyz.qwewqa.relive.simulator.core.stage.buff.*
+import xyz.qwewqa.relive.simulator.core.stage.Act
+import xyz.qwewqa.relive.simulator.core.stage.dress.DressCategory
+import xyz.qwewqa.relive.simulator.core.stage.dress.blueprint
+import xyz.qwewqa.relive.simulator.core.stage.passive.*
+
+val LittlePrinceMisora = dress3020009(
+    name = "Little Prince Misora",
+    acts = listOf(
+        ActType.Act1.blueprint("Strong Thrust") {
+            Act {
+                targetFront().act {
+                    attack(values1)
+                }
+            }
+        },
+        ActType.Act2.blueprint("Snake Bite") {
+            Act {
+                targetFront().act {
+                    attack(values1)
+                }
+                targetAllyAoe().act {
+                    applyBuff(DexterityUpBuff, values2, times2,)
+                }
+            }
+        },
+        ActType.Act3.blueprint("Fox Guidance") {
+            Act {
+                targetFront().act {
+                    attack(values1)
+                }
+                targetAllyAoe().act {
+                    applyBuff(CounterHealBuff, values2, times2)
+                }
+            }
+        },
+        ActType.ClimaxAct.blueprint("End of the Journey") {
+            Act {
+                targetAoe().act {
+                    attack(values1)
```

Comment:
Hit count is incorrect. As a general style rule, all acts should list a hit count for attacks even if the hit count is 1.